w10h02/test/pgdp/teams/Lib.java

```diff
@@ -0,0 +1,103 @@
+package pgdp.teams;
+
+import java.lang.reflect.Field;
+import java.util.Set;
+
+public class Lib {
+    @SuppressWarnings("")
+    public static Set<Penguin> getField(Lineup lineup, String fieldName)
+            throws NoSuchFieldException, IllegalAccessException {
+        Field field = lineup.getClass().getDeclaredField(fieldName);
+        field.setAccessible(true);
+        Set<Penguin> ret = (Set<Penguin>) field.get(lineup);
+        return ret;
+    }
+
+    //
+    // Disclaimer: Ähnlichkeiten zu realen Pinguinen sind rein zufällig.
+    //
+
+    // Artemis Example 1
+    public static Penguin jonas = new Penguin("Jonas", 10, 0, 0);
+    public static Penguin anatoly = new Penguin("Anatoly", 10, 10, 0);
+    public static Penguin julian = new Penguin("Juilan", 10, 10, 0);
```

Comment:
Did you mean `Julian`?

Reply:
No, I’m following Artemis Typos. Blame not me.